ablateLibrary/flow/fvFlow.cpp

```diff
@@ -274,7 +274,7 @@ void ablate::flow::FVFlow::ComputeTimeStep(TS ts, ablate::flow::Flow& flow) {
     }
 
     // take the min across all ranks
-    PetscInt rank;
+    int rank;
```

Comment:
We should probably use PetscMPIInt, to be consistent.

Reply:
Thanks!